src/rules/enforceLineBreak.js

```diff
@@ -0,0 +1,59 @@
+const schema = [];
+
+const create = (context) => {
+  return {
+    TypeAlias (node) {
+      const sourceCode = context.getSourceCode();
+      if (sourceCode.lines.length === 1) {
+        return;
+      }
+
+      if (node.loc.start.line !== 1) {
```

Comment:
Just curious what does this do?

Reply:
This is so you won't get the error 'New line required above type declaration` if the type declaration happens to be on the first line.